modules/gridNMBidAdapter.js

```diff
@@ -174,9 +181,27 @@ export const spec = {
         user.ext = userExt;
       }
 
+      const ortb2UserData = deepAccess(bidderRequest, 'ortb2.user.data');
```

Comment:
You should, at least in intent, be able to just do `request = mergeDeep({}, bidderRequest.ortb2, request)` instead of merging it piece by piece - if you are intentionally avoiding that I'm interested in understanding why. LGTM otherwise.

Reply:
@TheMediaGrid (just tagging for the question above)